src/type_set.rs

```diff
@@ -168,10 +171,12 @@ where
     }
 }
 
-fn _clone_test() {
+#[cfg(test)]
+#[test]
+fn clone_test() {
     fn is_clone<T: Clone>() {}
 
-    type T0 = <(String, u32) as TypeSet>::Variants;
+    type T0 = <(u8, u32) as TypeSet>::Variants;
```

Comment:
Having a test case that contains a `!Copy` type might be beneficial, can we use `alloc::string::String`?